python/revng/support/__init__.py

```diff
@@ -87,3 +89,29 @@ def get_command(command: str, search_prefixes: Iterable[str]) -> str:
 
 def get_llvmcpy():
     return LLVMCPy(_get_command("llvm-config", [get_root()]))
+
+
+WrappableInput = Union[str, bytes, IO[str], IO[bytes], mmap.mmap]
+
+
+@contextmanager
+def wrap_input(input_: WrappableInput) -> Generator[bytes, None, None]:
```

Comment:
`to_bytes`?

I find the usage of wrap confusing. I'm not sure in what sense we're "wrapping" `IO[str]`.

I'd avoid the "wrap" term entirely.

Reply:
Renamed to `to_bytes`